tensorboard/webapp/metrics/store/metrics_store_internal_utils_test.ts

```diff
@@ -1278,4 +1279,102 @@ describe('metrics store utils', () => {
       expect(getCardSelectionStateToBoolean(undefined, false)).toBeFalse();
     });
   });
+
+  describe('cardRangeSelectionEnabled', () => {
+    it('returns card specific value when defined', () => {
+      let cardStateMap = {
+        card1: {
+          rangeSelectionOverride: CardFeatureOverride.OVERRIDE_AS_ENABLED,
+        },
+      };
+      let globalRangeSelectionEnabled = false;
+      let linkedTimeEnabled = false;
+      let cardId = 'card1';
+
+      expect(
+        cardRangeSelectionEnabled(
+          cardStateMap,
+          globalRangeSelectionEnabled,
+          linkedTimeEnabled,
+          cardId
+        )
+      ).toBeTrue();
+      cardStateMap = {
+        card1: {
+          rangeSelectionOverride: CardFeatureOverride.OVERRIDE_AS_DISABLED,
+        },
+      };
+      globalRangeSelectionEnabled = true;
+      expect(
```

Comment:
nit: We have two expect here, let's add a new line between them. Otherwise it's too cluttered.
It's also a good thing to know the "arrange act assert" structure for unit tests.

Reply:
Makes sense, thanks!